### PR TITLE
GRW-1707 / Feature / Track offer_created event

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

Quick and dirty support for offer_created event tracking
- Send event via GTM
- Include very basic metadata
- Some TODOs for future improvement of tracking

Screenshot of event from GTM test page:
Screenshot 2022-10-31 at 12 57 29

## Justify why they are needed

We want to track conversion somehow when A/B testing, this adds first conversion event

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

File: apps/store/src/components/PriceForm/PriceForm.tsx [hunk 1]
```diff
@@ -21,11 +23,20 @@ type Props = {
...
     onCompleted(data) {
       const updatedPriceIntent = data.priceIntentConfirm.priceIntent
       if (updatedPriceIntent) {
+        // FIXME: pick offer for specific product or track all offers
+        const firstOffer = updatedPriceIntent.offers[0]
+        trackOffer({
+          shopSessionId: shopSession!.id,
+          contractType: firstOffer.variant.typeOfContract,
+          amount: firstOffer.price.amount,
+          currency: firstOffer.price.currencyCode,
```
Comment (gustaveen) [inline apps/store/src/components/PriceForm/PriceForm.tsx:38]: nit: do we want to send the offer as a single payload and extract the data in the trackOffer function instead? Guess it comes down to if we we reuse this in more places I guess

**Review by gustaveen — APPROVED** [review]
Great first step! 🙌

File: apps/store/src/components/PriceForm/PriceForm.tsx (hunk 1 above)
Comment (alebedev) [reply apps/store/src/components/PriceForm/PriceForm.tsx:38]: There's a place for smart tracking service between UI and GTM service. Agree that UI should not know which details we're using in tracking

**Review by amandaflod — APPROVED** [review]
Looks good to me! Great progress for tracking in hedvig.com 👍

**Pull request merged**